tests/FixedArrayTest.php

```diff
@@ -166,6 +166,61 @@ public function providerShiftDown() : array
         ];
     }
 
+    public function testCreate()
+    {
+        $fixedArray = FixedArray::create(4);
+
+        $this->assertInstanceOf(FixedArray::class, $fixedArray);
+        $this->assertSame(4, $fixedArray->getSize());
+    }
+
+    public function testToArray()
+    {
+        $expectedArray = [1, 2, 3, 4];
+        $fixedArray = FixedArray::fromArray($expectedArray);
+        $result = $fixedArray->toArray();
+
+        $this->assertCount(4, $result);
+        $this->assertSame($expectedArray, $result);
+    }
+
+    public function testGetSize()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+
+        $this->assertSame(4, $fixedArray->getSize());
+    }
+
+    public function testSetSize()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+        $fixedArray->setSize(5);
+
+        $this->assertSame(5, $fixedArray->getSize());
+    }
+
+    public function testOffsetExistsShouldReturnTrue()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+
+        $this->assertTrue($fixedArray->offsetExists(0));
+    }
+
+    public function testOffsetExistsShouldReturnFalse()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+
+        $this->assertFalse($fixedArray->offsetExists(5));
+    }
```

Comment:
For the 2 methods above, you can make a single function, `testOffsetExists()`, with a data provider:

```
[0, true],
[5, false]
```